### DISCUSSION TITLE
Domotizzare tapparelle da zero - consigli su hardware

### DISCUSSION THREAD

**Discussion by fregol** [indomus/forum#2951] [Hardware & Firmware]
Ciao! Sono abbastanza nuovo di HA, ma pian piano tutto sta prendendo sempre più senso!

Vorrei motorizzare da zero e quindi domotizzare le tapparelle di casa e vorrei qualche consiglio. 
L'idea di base è questa: fare arrivare la corrente nei cassonetti, installare il motore nei rulli e collegarli ad uno shelly 2PM plus (mi interessano sopratutto alcune automazioni per la mattina e la sera e la possibilità di comandare anche via alexa l'apertura, anche a metà e la chiusura di ogni tapparella). 
Dopodichè, un paio di tapparelle le cablerò per avere i tasti sali e scendi (vorrei usare VIMAR 14062), altre tapparelle dovranno avere tasti wireless (ho visto aqara, ma non sono sicuro).
Il consiglio che vi chiedo per avere piena integrazione su home assistant è proprio relativo a cosa acquistare esattamente (mantenendo un budget medio basso):
- motore INDEM MT30 (o MT 40, vedi amazon)
-  shelly 2PM Plus
- pulsanti cablati VIMAR 14062
- pusanti wireless ??

Mi dareste per favore qualche parere, e consigliare se con questo hardware funzionerà tutto liscio? 
Home assistant installato su raspberry e Sonoff dongle zigbee

**Comment by VincenzoMaletta** [comment]
Ciao,

non conosco nello specifico gli shelly 2PM Plus ma per le mie tapparelle utilizzo da anni due shelly 2.5 e funzionano egregiamente; puoi controllarli direttamente tramite i classici pulsanti interconnessi, tramite la loro app e tramite HA. Da quanto so sono gli unici che gestiscono correttamente l'apertura o chiusura parziale, specificando la percentuale desiderata. Molti dispositivi wireless per il controllo delle tapparelle non prevedono tempi diversi per l'apertura e la chiusura e quindi ti ritrovi con avvolgibili parzialmente chiusi non non completamente aperti. Ci sono anche i pulsanti zigbee della BTicino (l4027c) ma mi sa che da HA non puoi controllare l'apertura/chiusura parziale; solamente tutto su o tutto giù.
